packages/flet_geolocator/lib/src/geolocator.dart

```diff
@@ -0,0 +1,95 @@
+import 'dart:async';
+
+import 'package:flet/flet.dart';
+import 'package:flutter/widgets.dart';
+import 'package:geolocator/geolocator.dart';
+import 'package:permission_handler/permission_handler.dart';
+
+class GeolocatorControl extends StatefulWidget {
+  final Control? parent;
+  final Control control;
+  final FletControlBackend backend;
+
+  const GeolocatorControl(
+      {super.key,
+      required this.parent,
+      required this.control,
+      required this.backend});
+
+  @override
+  State<GeolocatorControl> createState() => _GeolocatorControlState();
+}
+
+class _GeolocatorControlState extends State<GeolocatorControl>
+    with FletStoreMixin {
+  @override
+  void initState() {
+    debugPrint("Geolocator.initState($hashCode)");
+    widget.control.onRemove.clear();
+    widget.control.onRemove.add(_onRemove);
+    super.initState();
+  }
+
+  void _onRemove() {
+    debugPrint("Geolocator.remove($hashCode)");
+    widget.backend.unsubscribeMethods(widget.control.id);
+  }
+
+  @override
+  void deactivate() {
+    debugPrint("Geolocator.deactivate($hashCode)");
+    super.deactivate();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    debugPrint(
+        "Geolocator build: ${widget.control.id} (${widget.control.hashCode})");
+    LocationAccuracy locationAccuracy = LocationAccuracy.best;
+
+    String locationAccuracyString =
+        widget.control.attrString("locationAccuracy", 'best')!;
+    switch (locationAccuracyString) {
+      case "best":
+        locationAccuracy = LocationAccuracy.best;
+      case "bestForNavigation":
+        locationAccuracy = LocationAccuracy.bestForNavigation;
+      case "high":
+        locationAccuracy = LocationAccuracy.high;
+      case "medium":
+        locationAccuracy = LocationAccuracy.medium;
+      case "low":
+        locationAccuracy = LocationAccuracy.low;
+      case "lowest":
+        locationAccuracy = LocationAccuracy.lowest;
+      case "reduced":
+        locationAccuracy = LocationAccuracy.reduced;
```

Comment:
Please move this to a utils file as seen here: https://github.com/flet-dev/flet/pull/3093/files#diff-2c856694204af9aa4aa908b108a4541b8dea385b5bb1f57870db9cdcc716c945

Then use it as seen here: https://github.com/flet-dev/flet/pull/3093/files#diff-a0b7d8d011def02618f1813be955bf63ad0b894bb238cb50c3087941d60027d8R54

Reply:
done